src/BD_sensor.h

```diff
@@ -0,0 +1 @@
+uint16_t Get_Distane_data(void);
```

Comment:
typo (distance)

Reply:
thanks, that header file has been deleted for it is unused now.